jbi/models.py

```diff
@@ -74,9 +74,27 @@ class ActionParams(BaseModel, frozen=True):
     jira_project_key: str
     steps: ActionSteps = ActionSteps()
     jira_components: JiraComponents = JiraComponents()
+    jira_cf_fx_points_field: str = "customfield_10037"
+    jira_severity_field: str = "customfield_10716"
+    jira_priority_field: str = "priority"
+    jira_resolution_field: str = "resolution"
     labels_brackets: Literal["yes", "no", "both"] = "no"
     status_map: dict[str, str] = {}
+    priority_map: dict[str, str] = {}
     resolution_map: dict[str, str] = {}
+    severity_map: dict[str, str] = {}
+    cf_fx_points_map: dict[str, int] = {
+        "?": 0,
+        "1": 1,
+        "2": 2,
+        "3": 3,
+        "5": 5,
+        "7": 7,
+        "8": 8,
+        "12": 12,
+        "13": 13,
+        "15": 15,
```

Comment:
I think I'm contractually obligated to call out that 7, 12, and 15 are not valid story point amounts.

Reply:
I just copied the dropdown values from Bugzilla UI:

<img width="188" alt="Screenshot 2024-03-06 at 10 25 28" src="https://github.com/mozilla/jira-bugzilla-integration/assets/546692/abef6554-dcdb-4bfb-9a9b-2f6271f9e24c">